testing/architecture-test/src/test/java/org/gradle/architecture/test/ArchUnitFixture.java

```diff
@@ -119,6 +120,19 @@ public boolean test(JavaClass javaClass) {
         .and(not(kotlin_internal_methods))
         .as("public API methods");
 
+    DescribedPredicate<JavaMethod> getters = new DescribedPredicate<JavaMethod>("getters") {
+        @Override
+        public boolean test(JavaMethod input) {
+            PropertyAccessorType accessorType = PropertyAccessorType.fromName(input.getName());
+            if (accessorType == PropertyAccessorType.IS_GETTER) {
```

Comment:
This should be fixed instead.

Reply:
@asodja

Reply:
Note: this code is only moved here, so I think it's okay to fix this in a separate PR.